stable/syncthing/2.1.0/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: syncthing
+version: 2.1.0
+upstream_version: 1.15
+appVersion: "auto"
+description: P2P file synchronization application
+type: application
+deprecated: false
+home: https://syncthing.net/
```

Comment:
One nitpick: The home of your chart is this repository, syncthing.net is your source ;-)